lib/v2/screens/sign_up/add_phone_number/add_phone_number.dart

```diff
@@ -33,37 +35,43 @@ class _AddPhoneNumberState extends State<AddPhoneNumber> {
         body: BlocConsumer<SignupBloc, SignupState>(
           listener: (context, state) {},
           builder: (context, state) {
-            return Padding(
-              padding: const EdgeInsets.all(16),
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: <Widget>[
-                  TextFormFieldCustom(
-                    labelText: "Add Phone Number (optional)".i18n,
-                    keyboardType: TextInputType.phone,
-                    controller: _keyController,
-                    inputFormatters: [
-                      phoneNumberInternationalFormatter,
+            return Stack(
```

Comment:
Stack not needed here, is a really small screen, also I suggest Scaffold is returned here. 
Also suggest to use scaffold (bottomSheet) to force button to go to the end of screen instead of using expanding for it.

Reply:
Stack is for loading screen to block the user from pressing the Create Account button twice or editing the phone number field when loading.